src/main/kubeconfig-manager.ts

```diff
@@ -72,8 +76,13 @@ export class KubeconfigManager {
     return tempFile;
   }
 
-  unlink() {
+  async unlink() {
```

Comment:
I could not find any references where we are using this code. Possible dead code?

Reply:
I think it might be dead code. Only `Cluster` uses the `KubeconfigManager` and it doesn't have a "cleanup" method.